templates/watcher/config/watcher-dbsync-config.json

```
@@ -0,0 +1,3 @@
+{
+    "command": "watcher-db-manage upgrade"
```

Based on https://docs.openstack.org/watcher/1.0.0/man/watcher-db-manage.html#upgrade and https://github.com/openstack-k8s-operators/neutron-operator/blob/a3ded0c987226692b66791d6556d6fc3d1ed84d7/templates/neutronapi/config/db-sync-config.json#L2 Do we also pass the config file to this code like this? Just a thought
```suggestion
    "command": "watcher-db-manage --config-file=/etc/watcher/watcher.conf upgrade"
```

---

> Based on https://docs.openstack.org/watcher/1.0.0/man/watcher-db-manage.html#upgrade and https://github.com/openstack-k8s-operators/neutron-operator/blob/a3ded0c987226692b66791d6556d6fc3d1ed84d7/templates/neutronapi/config/db-sync-config.json#L2 Do we also pass the config file to this code like this? Just a thought

Based on this https://docs.openstack.org/watcher/1.0.0/man/watcher-db-manage.html#usage.
```
If no configuration file is specified with the [--config-file](https://docs.openstack.org/watcher/1.0.0/man/watcher-db-manage.html#cmdoption-watcher-db-manage--config-file) option, watcher-db-manage assumes an SQLite database.
```
Not sure we need it or not. Need your thoughts on this.

---

Actually, we don't need it, default values are:

config_file                    = ['/etc/watcher/watcher.conf']
config_dir                     = ['/etc/watcher/watcher.conf.d']

in this case, in fact we are copying the real config file under '/etc/watcher/watcher.conf.d', so I'd prefert to just rely on defaults.

---

Thank you for explaining it! Make sense!